### DISCUSSION TITLE
Milestones section looks a bit unmaintained

### DISCUSSION THREAD

**Discussion by usbeehu** [pop-os/cosmic-epoch#1386] [General]
Alpha 3 still in unfinished state and Alpha 4 is missing entirely. I understand that this is not a priority during this phase of development but still I'd like to actively follow the current state of development and also to see a rough estimation for the release date for the next alpha releases. Thank you, I wish you guys the best for the future and also wish you all a happy and resting holiday!

**Comment by mmstick** [comment]
You're looking in the wrong place. The COSMIC board is here: https://github.com/orgs/pop-os/projects/23
Outside of that, each github repository has its own milestones for each alpha.

**Reply by usbeehu** [reply]
Thank you for pointing this out!
